docs/moment/00-use-it/10-system-js.md

```diff
@@ -0,0 +1,42 @@
+---
+title: System.js
+---
+
+To load moment, place it in the path specified by your System.config in the baseURL configuration. 
+Then import it into your page.
+
+```js
+<script src="system.js"></script>
+<script>
+  System.config({
+    baseURL: '/app'
+  });
+  
+  System.import('moment.js');
+ </script>
+ ```
+
+
+If you need moment to be loaded as global, you can do this with the meta configuration:
+
+```javascript
+System.config({
+  meta: {
+    'moment': { format: 'global' }
+  }
+});
+```
+
+Alternatively, if Moment isn't running as a global, you can configure moment to be a global for the global that is running:
```

Comment:
I thought that's what the first option did. Looking at the SystemJS docs, I think it might be that if you have some other dependency (in this case `path/to/global-file.js`) that itself depends on having `moment` defined as a global, you do it this second way. (See here: https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md#globals). If that's true (a big if), I think the language that would be more useful would be:

> Alternatively, to provide Moment as a global to a specific dependency, you can do this: